src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+---
+title: EKS Fargate
+watermark: BETA
+---
+
+In order to monitor Kubernetes workloads scheduled on Fargate, our solution automatically injects a sidecar containing the infrastructure agent and the nri-kubernetes integration in each pod that needs to be monitored.
+
+If the same Kubernetes cluster also contains EC2 nodes, our solution is deployed as a DaemonSet in all of them. No sidecar will be injected into pods scheduled in EC2 nodes, and no DaemonSet will be deployed to Fargate nodes. Here's an example of a hybrid instance with both Fargate and EC2 nodes:
+
+![Diagram showing an EKS cluster with Fargat and EC2 nodes](./images/fargate-overview.svg "Diagram showing an EKS cluster with Fargat and EC2 nodes")
+<figcaption>
+In this mixed environment, the integration only uses a sidecar for Fargate nodes.
+</figcaption>
+
+New Relic collects all the supported metrics for all Kubernetes objects regardless of where they are scheduled, whether it's Fargate or EC2 nodes. Please note that, due to Fargate limitations, the New Relic integration is limited to running in [unprivileged](/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure/#unprivileged) mode in Fargate nodes. This means that metrics that are usually fetched from the host directly, like running processes, will not be available for Fargate nodes.
+
+The agent in both scenarios will scrape data from Kube State Metrics (KSM), Kubelet and cAdvisor and send data in the same format.
+
+<Callout variant="important">
+You are still required to deploy and monitor a Kube State Metrics instance, which the Helm Chart can do automatically if instructed to. KSM will be monitored as any other workload: by injecting a sidecar if it gets scheduled in a Fargate node, or with the local instance of the DaemonSet if it gets scheduled on an EC2 node.
+</Callout>
+Other components such as `nri-prometheus`, `nri-metadata-injection`, and `nri-kube-events` do not need any extra steps to be deployed on Fargate.
+
+<Callout variant="tip">
+New Relic Logging is not yet supported on Fargate nodes.
+</Callout>
+
+## Installation [#installation]
+
+You can choose between two alternatives for installing New Relic full observability in your EKS Fargate cluster: 
+
+* [Automatic (recommended)](#automatic)
+* [Manual injection](#manual)
+
+Regardless of the approach you choose, the experience is exactly the same after it is installed. The only difference is how the container is injected.
+
+### Automatic injection (recommended) [#automatic]
+
+By default, when Fargate support is enabled, New Relic will deploy an operator to the cluster. (newrelic-infra-operator). Once deployed, this operator will automatically inject the monitoring sidecar to pods that are scheduled into Fargate nodes, while also managing the creation and the update of Secrets, ClusterRoleBindings, and any other related resource.
```

Comment:
I don't know about the conventions in this repository, but it's often useful to manually add line-breaks in markdown, so it's easier spot changes/suggestions and to make text better readable in terminals with limited width (same rule as for code).

It also makes it easier to read on GitHub.

Reply:
@invidian, thanks for the suggestion! We haven't been following that convention in our docs, but I'll discuss this approach with the team.